Components/NavigationMain/_style.scss

```diff
@@ -22,11 +22,11 @@
 
   .logo {
     display: block;
-    flex-shrink: 0;
```

Comment:
Had to remove this line, because IE was stretching big images uploaded via customizer like this:
![Screen Shot 2020-06-08 at 13 59 03](https://user-images.githubusercontent.com/44668081/84029191-48356680-a992-11ea-9f7d-2142c8d7a078.jpg)